conf/quincy/cephadm/sanity-cephadm.yaml

```diff
@@ -0,0 +1,68 @@
+globals:
```

Comment:
Update config name to `1admin-8node-1client-cluster.yaml`

Reply:
done